tests/tensorflow2/test_keras.py

```diff
@@ -361,11 +356,10 @@ def test_gradtape_hook_from_json(out_dir, monkeypatch):
 
     trial = smd.create_trial(path=out_dir)
     # can't save gradients in TF 2.x
-    assert len(trial.tensor_names()) == 4
+    assert len(trial.tensor_names()) == 3  # 4
```

Comment:
What's happening to metrics? Looks like those are no longer being recorded.

Reply:
Metrics wasn't getting exported as export_collection was called in pop_tape() before user could manually save metrics tensor.
Modified the code to behave similar to PT hook - export_collections in push_tape and introduce record_tensor_value